FreeRTOS/FreeRTOS-Plus-TCP
# Fixing GCC compiler warning

Fixing GCC compiler warning

Description
-----------
This allows compilation of the socket source in projects using GCC with -pedantic= and -Werror. The change has no functional impact.

Test Steps
-----------
Compiles with GCC, -pedantic, and -Werror

Checklist:
----------
- [x] I have tested by changes but have not run the existing tests.

Related Issue
-----------
(none)


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
